### DISCUSSION TITLE
remove this feature request template or update the template

### DISCUSSION THREAD

**Discussion by shoulders** [joomla/joomla-cms#31748] [General]
if all new features are being moved to the ideas section of discussions (which I did not know existed) then you should remove this template or update it to reflect the new policy.

thanks

**Comment by ghost** [comment]
duplicate report: https://github.com/joomla/joomla-cms/discussions/31705 please close.

**Comment by shoulders** [comment]
This is not a duplicate. The feature template is not even mentioned in #31705

I am not sure what they are even talking about either but is not the template.

**Comment by infograf768** [comment]
@shoulders 
What kind of changes would you suggest?

**Comment by chmst** [comment]
@shoulders not every feature request is automatically a discussion. We always can decide to keep a feature request here in issues.

**Comment by shoulders** [comment]
@infograf768 
"Dont post feature requests here, instead create a discussion in the category `ideas` [here](https://github.com/joomla/joomla-cms/discussions/new)"

@chmst my valid feature requests have been closed and  moved to ideas
- https://github.com/joomla/joomla-cms/discussions/31253
- https://github.com/joomla/joomla-cms/discussions/27810

These dont need discussions, they are what they are and I dont know why they were moved. Personally I liked it when joomla had ideas.joomla.org

**Comment by brianteeman** [comment]
ideas.joomla.org was never looked at. It was over a year before anyone even noticed that you couldnt access it

**Comment by brianteeman** [comment]
> if all new features are being moved to the ideas section of discussions (which I did know existed)

Thats because the production department dont consider it important to communicate with the community or to report on their meetings

**Comment by shoulders** [comment]
on a side note, I do find the joomla real estate fragmented i.e. multiple blogs all over the place. `1 blog to rule them all` is better 😄

**Comment by ghost** [comment]
@shoulders sorry for misunderstanding your first comment.

**Comment by zero-24** [comment]
> > if all new features are being moved to the ideas section of discussions (which I did know existed)
> 
> Thats because the production department dont consider it important to communicate with the community or to report on their meetings

Thats a bold statment and as a member of the production department i have to strongly say that this is not true. Yes we are lacking behind with the reports but this is beeing worked on already (or have been fixed already).

The meeting reports are always created but, please dont ask me why, where not pushed to the volunteer portal.

But lucky us we have more than the meeting reports as communication channel ;)

> if all new features are being moved to the ideas section of discussions (which I did not know existed)

It where just enabled a at the 18.12 so a few days before: https://developer.joomla.org/news/835-enabling-github-discussion-for-the-main-joomla-repository.html

**Comment by brianteeman** [comment]
> But lucky us we have more than the meeting reports as communication channel ;)

ah - the fictional power of telepathy

**Comment by zero-24** [comment]
Well there is Glip, GitHub, Twitter, joomla.org Sites, eMail, the magazine?

But this is going to be off-topic.

**Comment by brianteeman** [comment]
six of those seven have nothing

**Comment by zero-24** [comment]
have nothing of what?

**Comment by ChristineWk** [comment]
> It where just enabled a at the 18.12 so a few days before: https://developer.joomla.org/news/835-enabling-github-discussion-for-the-main-joomla-repository.html

Thank you for above link. I also didn't know about the interesting discussion section: 
https://github.com/joomla/joomla-cms/discussions

Now I know, that Core Update 3.9.24 is postponed :-) Usually I'm checking always: Milestone URL for date/info.

**Comment by shoulders** [comment]
why has this now been moved to discussions when it clearly is a bug.

The github feature template either needs to be removed or updated to follow these new guidelines, a discussion has apparently already been done for this issue. 😄 

Some groups at joomla do seem to live in their own bubble.
